apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx

```diff
@@ -133,7 +134,52 @@ const ColumnEditor = ({
 
   if (!columnFields) return null
 
+  const setPlacholderText = (format: string | undefined, columnFieldName: string) => {
+    const columnName = Boolean(columnFieldName) === true ? columnFieldName : "column_name"
+    switch (format) {
+      case "int2":
+      case "int4":
+      case "int8":
+      case "numeric":
+        setPlaceholder(`${columnName} > 0`)
+        break
+      case "float4":
+      case "float8":
+        setPlaceholder(`${columnName} > 0.0`)
+        break
+      case "text":
+      case "varchar":
+        setPlaceholder(`LENGTH(${columnName}) <= 50`)
+        break;
+      case "json":
+      case "jsonb":
+        setPlaceholder(`JSONB_TYPEOF(settings->'active') = 'boolean'`)
```

Comment:
Should this be something like:
```suggestion
        setPlaceholder(`JSONB_TYPEOF(${columnName}->'active') = 'boolean'`)
```
?

Reply:
You're right, I've updated this field.